Directory.Build.props

```diff
@@ -20,7 +20,7 @@
     <Platform Condition="'$(Platform)' == '' AND '$(BuildArchitecture)' == 'arm64'">$(BuildArchitecture)</Platform>
     <Platform Condition="'$(Platform)' == ''">x64</Platform>
 
-    <UseStableVersions Condition="'$(UseStableVersions)' == ''">true</UseStableVersions>
+    <UseStableVersions Condition="'$(UseStableVersions)' == ''">false</UseStableVersions>
```

Comment:
Should probably go away entirely, but this is already tracked at https://github.com/dotnet/source-build/issues/1627. (Noting for backlink and context on this PR.)

Reply:
Agreed